software/bunkerweb.yml

```diff
@@ -0,0 +1,15 @@
+name: "BunkerWeb"
+website_url: "https://www.bunkerweb.io"
+source_code_url: "https://github.com/bunkerity/bunkerweb"
+description: "Next-gen Web Application Firewall (WAF) that will protect your web services."
+licenses:
+  - AGPL-3.0
+platforms:
+  - deb
+  - Docker
+  - K8S
+  - Shell
+tags:
+  - Proxy
```

Comment:
To be fair this should probably go to https://github.com/awesome-foss/awesome-sysadmin?tab=readme-ov-file#web which lists Web servers and Reverse Proxies. The proxy section is for "traditional", outgoing proxies.

I'd like to do some reorganization (possibly move Web Servers/Reverse Proxies back to this list) before merging this.

Reply:
Hello @nodiscc,

Thanks for the feedback. I can create a PR on awesome-sysadmin if you prefer ?

Reply:
I will take care of the reorganization and merge this when I'm done. Assigning this to myself.